src/shortcuts/kodiak/honey-usdc.ts

```diff
@@ -36,13 +36,14 @@ export class KodiakHoneyUsdcShortcut implements Shortcut {
       tokensOut: [island],
     });
     const amountIn = builder.add(balanceOf(usdc, walletAddress()));
-    const halfAmount = div(amountIn, 2, builder);
-    const mintedAmount = await mintHoney(usdc, halfAmount, builder);
```

Comment:
```
    const usdcToMintHoney = getSetterValue(builder, this.setterInputs[chainId], 'usdcToMintHoney');
    const remainingUsdc = sub(amountIn, usdcToMintHoney, builder);
    const mintedAmount = await mintHoney(usdc, usdcToMintHoney, builder);
```

Shouldn't these changes need to be applied to other shortcuts that mint honey from usdc and deposit to kodiak too?

Reply:
the only other kodiak vault that mints honey is honey-weth. it is assumed that the usdc that is passed is in the correct ratio to the weth that is passed